app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt

```diff
@@ -28,29 +28,38 @@ import android.view.ViewGroup
 import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.SearchView
 import androidx.appcompat.widget.Toolbar
+import androidx.lifecycle.ViewModelProvider
 import org.kiwix.kiwixmobile.R
 import org.kiwix.kiwixmobile.cachedComponent
 import org.kiwix.kiwixmobile.core.base.BaseActivity
+import org.kiwix.kiwixmobile.core.base.BaseFragment
 import org.kiwix.kiwixmobile.core.base.FragmentActivityExtensions
 import org.kiwix.kiwixmobile.core.extensions.ActivityExtensions.start
+import org.kiwix.kiwixmobile.core.extensions.ActivityExtensions.viewModel
 import org.kiwix.kiwixmobile.core.main.CoreMainActivity
 import org.kiwix.kiwixmobile.core.utils.SimpleTextListener
 import org.kiwix.kiwixmobile.language.LanguageActivity
-import org.kiwix.kiwixmobile.zim_manager.library_view.LibraryFragment
+import org.kiwix.kiwixmobile.zim_manager.ZimManageViewModel
+import javax.inject.Inject
 
-class OnlineLibraryFragment : LibraryFragment(), FragmentActivityExtensions {
+class OnlineLibraryFragment : BaseFragment(), FragmentActivityExtensions {
```

Comment:
I don't think this is what was intended from the ticket.

The idea was there shouldn't be 2 classes when there is only 1 actually used type. LibraryFragment & OnlineLibraryFragment should become one class

Reply:
So ain't we going to delete `LibraryFragment` and put its required content appropriately in `OnlineLibFragment`?

Reply:
We are going to do it, this contributor doesn't have the time to